pkg/extractor/extractor.go

```diff
@@ -69,33 +71,49 @@ type FileConfigurations struct {
 
 func ParseYaml(content string) (*[]Configuration, error) {
 	configurations, err := extractYamlConfigurations(content)
-	if err != nil {
+	if err != nil && err != io.EOF {
 		return nil, err
 	} else {
 		return configurations, nil
 	}
 }
 
+func ParseJsonToK8sValues(content []byte) (map[string]interface{}, error) {
```

Comment:
I think the name is a little odd. It's not necessarily JSON it's basically any byte array and it's unclear what is `k8sValues`.